cou/utils/__init__.py

```diff
@@ -14,10 +14,61 @@
 
 """Utilities for charmed-openstack-upgrader."""
 
+import inspect
+import logging
 import os
 from pathlib import Path
+from typing import Any
 
+from aioconsole import ainput
 from halo import Halo
 
+from cou.utils.text_styler import bold, normal
+
 COU_DATA = Path(f"/home/{os.getenv('USER')}/.local/share/cou") if os.getenv("USER") else Path(".")
 progress_indicator = Halo(spinner="line", placement="right")
+
+
+def print_and_debug(message: Any) -> None:
+    """Print and log message at debug level.
+
+    :param message: The object to print and log.
+    :type message: Any
+    """
+    print(message)
+
+    logger = logging.getLogger(inspect.stack()[1].filename)
+    logger.debug(message)
+
+
+# pylint: disable=dangerous-default-value
```

Comment:
You should not use dangerous default valu, so I will suggest using None instead of list of "y" and "n".